### DISCUSSION TITLE
Advantages of this software over CAN communication.

### DISCUSSION THREAD

**Discussion by artur1702** [Louisvdw/dbus-serialbattery#1141] [Q&A]
Hello,

I have Multiplus II and JK BMS PB2A16S20P with latest software. They are currently connected with a CAN cable. Everything seems to be working properly.

Will I gain anything more by using this software and a USB->RS485 cable? What are the advantages of using this software compared to CAN communication?

Thanks,
Artur

**Comment by mattmeye** [comment]
I ask myself exactly the same question. I'm still in the planning as I have a similar setup and want to replace my temporary Pylontech US2000C.
Do you have multiple packages, or do you know if it works with 3 packages and CAN-BUS?

**Reply by artur1702** [reply]
At the moment I only use one battery package.

**Reply by entropia1ac** [reply]
1) Via CAN there is only a preview of cell Vmax and Vmin, via RS485 you can see the voltage of each cell.
 2) Via CAN jkBms controls the charging and via RS485 Victron does it.

**Reply by artur1702** [reply]
Thank you for the explanation
